napari/_qt/qt_viewer.py

```diff
@@ -329,6 +331,16 @@ def __init__(self, viewer: ViewerModel, show_welcome_screen: bool = False):
         # bind shortcuts stored in settings last.
         self._bind_shortcuts()
 
+    def _leave_canvas(self):
+        """disable status on canvas leave"""
+        self.viewer.status = ""
+        self.viewer._mouse_over_canvas = False
+
+    def _enter_canvas(self):
+        """enable status on canvas enter"""
+        self.viewer.status = "Ready"
```

Comment:
Optional: I'm not convinced how useful this is, so I'd consider removing it. But I don't have strong feelings.

If you do want to keep it then I think it should be a translated string right?

Reply:
Here is also a lack of translation. Should I replace both places for consistency?

https://github.com/napari/napari/blob/f3289ac3f5480706afa456ad8965c8b4336c4c9d/napari/components/viewer_model.py#L129

Reply:
Ah that's where this comes from! Thanks for the pointer!

I think my dream would be to define a constant in `viewer_model.py` like `STATUS_DEFAULT = trans._('Ready')`, then use that in both places. What do you think? Even you like it, definitely feel free to write up an issue or do that in a separate PR if you want to get this one in without another change.